### ISSUE TITLE
-mlir-print-op-generic does not appear in 'mlir-opt -h'  help message

### ISSUE THREAD

**Issue by bondhugula** [tensorflow/mlir#34]
The option -mlir-print-op-generic does not appear in the output of 'mlir-opt -h'. mlir-opt does accept it though.

**Comment by lattner** [comment]
Does it show up in the --help-hidden output?

**Comment by Johnson9009** [comment]
Yes, just as lattner said, this option is an hidden option from the code.
https://github.com/tensorflow/mlir/blob/43aa628b747f253228b3628ecbcc3d0df03fb2af/lib/IR/AsmPrinter.cpp#L79

**Comment by bondhugula** [comment]
Thanks. Could we unhide it? It's at least more useful for developers than many of the options currently exposed are.
